docs/src/pages/de/installation.md

```diff
@@ -29,65 +29,65 @@ yarn create astro
 pnpm create astro
 ```
 
-Mit dem [`create-astro`](https://github.com/snowpackjs/astro/tree/main/packages/create-astro) Assistenten kannst du aus einer Reihe von [Starter-Vorlagen](https://github.com/snowpackjs/astro/tree/main/examples) wählen. Alternativ kannst du auch dein eigenes Astro-Projekt direkt von GitHub importieren.
+Der [`create-astro`](https://github.com/snowpackjs/astro/tree/main/packages/create-astro)-Assistent lässt dich aus einer Reihe von [Start-Vorlagen](https://github.com/snowpackjs/astro/tree/main/examples) auswählen. Alternativ könntest du auch dein eigenes Astro-Projekt direkt aus GitHub heraus importieren.
 
 ```bash
-# Hinweis: Ersetze "my-astro-project" durch den Namen deines Projekts.
+# Hinweis: Ersetze "mein-astro-projekt" durch den Namen deines Projekts.
 
 # npm 6.x
-npm init astro my-astro-project --template starter
+npm init astro mein-astro-projekt --template starter
 # npm 7+ (zusätzliche Bindestriche sind erforderlich)
-npm init astro my-astro-project -- --template starter
+npm init astro mein-astro-projekt -- --template starter
 # yarn
-yarn create astro my-astro-project --template starter
+yarn create astro mein-astro-projekt --template starter
 # pnpm
-pnpm create astro my-astro-project -- --template starter
+pnpm create astro mein-astro-projekt -- --template starter
 # Verwenden einer Drittanbietervorlage
-npm init astro my-astro-project -- --template [GITHUB_USER]/[REPO_NAME]
-# Verwenden einer Drittanbietervorlage innerhalb eines repos
-npm init astro my-astro-project -- --template [GITHUB_USER]/[REPO_NAME]/path/to/template
+npm init astro mein-astro-projekt -- --template [GITHUB_NAME]/[REPO_NAME]
+# Verwenden einer Drittanbietervorlage innerhalb eines Repos
+npm init astro mein-astro-projekt -- --template [GITHUB_NAME]/[REPO_NAME]/pfad/zur/vorlage
 ```
 
-Nachdem `create-astro` dein Projekt vorbereitet hat, denke daran, die Projektabhängigkeiten mit npm oder dem Paketmanager deiner Wahl zu installieren. In diesem Beispiel verwenden wir npm:
+Nachdem `create-astro` dein Projekt eingerichtet hat, denke daran die Abhängigkeiten deines Projekts mittels npm oder dem Paketmanager deiner Wahl zu installieren. In diesem Beispiel verwenden wir npm:
 
 ```bash
 npm install
 ```
 
-Du kannst dein Astro-Projekt nun [Starten](#start-astro). Nachdem du dein Astro-Projekt fertiggestellt hast, kannst du dein Projekt [Bauen](#build-astro). Astro wird dann die statischen Seiten generieren und für dich bereitstellen, welche du anschließend bei deinem bevorzugten Hosting-Anbieter [Ausrollen](/guides/deploy) kannst.
+Du kannst dein Astro-Projekt nun [starten](#start-astro). Sobald du dein Astro-Projekt fertiggestellt hast, kannst du dein Projekt [kompilieren](#build-astro). Astro wird dann deine Anwendung fertig packen und dafür die statischen Seiten generieren, die du bei deinem bevorzugten Hosting-Anbieter [ausrollen](/guides/deploy) kannst.
```

Comment:
```suggestion
Du kannst dein Astro-Projekt nun [starten](#start-astro). Sobald du dein Astro-Projekt fertiggestellt hast, kannst du dein Projekt [kompilieren](#build-astro). Astro wird dann deine Anwendung fertig packen und dafür die statischen Seiten generieren, die du bei deinem bevorzugten Hosting-Anbieter [veröffentlichen](/guides/deploy) kannst.
```
`ausrollen` is not something we say in that context in Germany really, even though it would probably work, it kinda sounds a bit awkward 😄
I'd suggest `veröffentlichen` as in `publish`.

Reply:
Yeah, I had a pretty bad time trying to accept this (it was in a previous version), and yes it is way over the top, and almost no one would actually use it. Thank you for throwing it out)))